modules/nw-kube-proxy-config.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * networking/configuring-kubeproxy.adoc
+
+[id="modifying-kubeproxy-config-{context}"]
+= kube-proxy configuration parameters
+
+You can modify the following `kubeProxyConfig` parameters:
+
+.Parameters
+[cols=".^2,.^5,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`bindAddress`
+|The IP address for `kube-proxy` to serve on.
+|The default value is `0.0.0.0`.
+
+|`iptablesSyncPeriod`
+|The refresh period for `iptables` rules. The default value is `30s`. Valid
+suffixes include `s`, `m`, and `h` and are described in the
+link:https://golang.org/pkg/time/#ParseDuration[Go time package] documentation.
+|`30s`
+
+|`proxyArguments.iptables-min-sync-period`
+|The minimum duration before refreshing `iptables` rules. This parameter ensures
+that the refresh does not happen too frequently. Valid suffixes include `s`,
+`m`, and `h` and are described in the
+link:https://golang.org/pkg/time/#ParseDuration[Go time package]
+|`30s`
+
+|`proxyArguments.proxy-mode`
+|The implmentation for synchronizing network rules.
```

Comment:
implementation

Please provide a little more info about these parameter values. I'm not sure why I'd pick one over another.

Reply:
No longer exposing this parameter.